### DISCUSSION TITLE
启动失败，有无大佬指点一下呜呜呜 Failed to start, is there any expert who can give me some advice?

### DISCUSSION THREAD

**Discussion by guapia233** [chatchat-space/Langchain-Chatchat#1913] [Q&A]
按照新版本一步步部署 启动报错 不知如何排查
![1698584823235](https://github.com/chatchat-space/Langchain-Chatchat/assets/66232990/0a7e925c-bb0b-4a92-b2d0-a11e57106e8b)
![1698584841559](https://github.com/chatchat-space/Langchain-Chatchat/assets/66232990/9fd5b84a-1d16-4c5c-ad3a-1949df5beb54)

**Comment by zRzRzRzRzRzRzR** [comment]
更新到最新代码，可以在微信群问一下
